src/ensembl/src/shared/components/autosuggest-search-field/AutosuggestSearchField.test.tsx

```diff
@@ -200,43 +206,46 @@ describe('<AutosuggestSearchField />', () => {
     });
 
     describe('when raw input submission is allowed', () => {
-      let mountedComponent: any;
-
-      beforeEach(() => {
-        mountedComponent = mount(
-          <AutosuggestSearchField
-            search={search}
-            onChange={onChange}
-            onSelect={onSelect}
-            matchGroups={groupsOfMatches}
-            allowRawInputSubmission={true}
-            onSubmit={onSubmit}
-          />
-        );
-      });
+      const props = {
+        search,
+        onChange,
+        onSelect,
+        matchGroups: groupsOfMatches,
```

Comment:
Would it be worth defining these as `commonProps`?

Reply:
✅